docs/_getting_started/jupyter.md

```diff
@@ -136,35 +136,19 @@ Threads.nthreads()
 The output of `Threads.nthreads()` should be equal to the number of threads you used to create the kernel.
 
 #### Stata
-* Load Stata module:
-
-    ```title="Terminal Command"
-    ml statamp/18
-    ```
-
-* Make a new Python `venv` environment, install Stata kernel, and add it to Jupyter:
-
-    ```title="Terminal Commands"
-    /usr/bin/python3 -m venv stata_kernel 
-    source stata_kernel/bin/activate
-    pip install stata_kernel jupyter 
-    python -m stata_kernel.install
-    ```
-
-* Before you test JupyterHub, check that the Stata kernel is using the correct Stata executable.  
-
-    ```title="Terminal Command"
-    cat ~/.stata_kernel.conf
-    ```
-    After you type the above in your terminal, you should see:
-
-    ```{.stata .yaml .no-copy title="Terminal Output"}
+* Install Stata kernel by running this command from a terminal:
+```title="Terminal Command"
+install_stata_kernel
+```
 
-    [stata_kernel]
-    
-    stata_path = /software/non-free/stata18/stata-mp
+    If everything worked, you should see:
```

Comment:
I would add a brief note about what this script is doing and importantly, that it will take several minutes to complete. Perhaps also add some of the earlier print statements that show up before all of the `pip` installs? Like

```
Loading Stata module...
Creating Python virtual environment for Stata kernel...
Activating virtual environment...
Installing Stata kernel and Jupyter...
```